pkg/util/glog/glog.go

```diff
@@ -0,0 +1,97 @@
+package glog
```

Comment:
nit picky but if this is going to grow into a generic log wrapper, i don't think we should be calling it glog.  it's going to make for confusing code (is this code using the wrapper or the raw glog?  have to go check the import to know)